java/core/src/java/org/apache/orc/OrcConf.java

```diff
@@ -329,6 +341,14 @@ public String getString(Configuration conf) {
     return getString(null, conf);
   }
 
+  public List<String> getStringAsList(Configuration conf) {
+    String confStr = getString(null, conf);
+    if (confStr == null || confStr.isEmpty()) {
+      return new ArrayList<>();
+    }
+    return Arrays.asList(confStr.split(","));
+  }
```

Comment:
If you don't mind, could you open a new JIRA and PR for this, @deshanxiao ?

And, it would be great to remove white space and empty strings after `split`. For example, `a,,b,c,`?

Reply:
I saw your JIRA and have been waiting for your PR, @deshanxiao  :)

Reply:
I merged https://github.com/apache/orc/pull/1337 .